BuildAndServe.bat

```diff
@@ -0,0 +1,3 @@
+.\docfx\docfx.exe docfx.json --serve
```

Comment:
The latest release is extracted to the docfx folder below the project root.
![capture44](https://cloud.githubusercontent.com/assets/140737/16928221/64d24db0-4ce6-11e6-8484-bd7ce16cd645.PNG)